builder.go

```diff
@@ -140,13 +140,40 @@ type Dependency struct {
 	Version string `json:"version,omitempty"`
 }
 
+// ReplacementPath represents an old or new path component within
+// a Go module replacement directive
+type ReplacementPath string
+
+// Param is a helper that reformats a go.mod replace directive
+// to be compatible with the go mod edit command
+func (r ReplacementPath) Param() string {
+	if r == "" {
+		return ""
+	}
+	return regexp.MustCompile(`\sv`).ReplaceAllString(string(r), "@v")
```

Comment:
I don't think there's any need for regular expressions, and we can't assume the version starts with `v` (it could be a commit or branch name):

```suggestion
	return strings.Replace(string(r), " ", "@", 1)
```

Reply:
From what I understand, the "version" here will always start with a `v`. See: https://golang.org/ref/mod#versions

Reply:
We can try it, but `go get` accepts commits and branches too.